docs/setup/mac.md

```diff
@@ -97,6 +97,6 @@ If VS Code doesn't update once it restarts, it might be set under quarantine by
 
 ### Does VS Code run on Mac M1 machines?
 
-Yes, VS Code supports macOS ARM64 builds that can run on Macs with the Apple M1 chip starting with version [1.54](/release-notes/v1_54.md) and [Insiders](/insiders) builds.
+Yes, VS Code supports macOS ARM64 builds that can run on Macs with the Apple M1 chip.
 
 ![macOS ARM64 Insiders build](images/mac/arm64-insiders.png)
```

Comment:
should we remove this image as well? The image isn't in the repo, but if you view the site [here](https://code.visualstudio.com/docs/setup/mac), it looks like that image is outdated since it shows "Intel" and "ARM64". It seems like the naming has changed to be "Intel Chip" and "Apple Silicon" (see image below from insiders download page):

![image](https://user-images.githubusercontent.com/29909203/110400953-7e8d3880-802d-11eb-8293-7f2e248077f1.png)